spec/latest/json-ld-framing/index.html

```diff
@@ -301,7 +305,10 @@
       <a href="../json-ld-api/#compaction">Compacting</a>.</dd>
     <dt><code>@omitDefault</code></dt>
     <dd>Used in <a href="#framing">Framing</a> to override the
-      value of <a>omit default flag</a> within a specific frame.</dd>
+      value of <a>omit default flag</a> within a specific frame.
+      <div class="issue">Is this flag really useful? Easier to simply not have a default value,
+        if it shouldn't match.</div>
```

Comment:
It's confusing naming. If a property value has `@omitDefault: true`, it will not generate a property with `null` in the output, or use the value of `@default`. Otherwise, we could just not put the `@default` in in the first place. An alternative would be to remove `@omitDefault` and _not_ add missing property values to the output without `@default`. We could then allow `@default: @null` to explicitly give a `null` value in the output.